CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 #### Changed
 - Upgraded to IDE Rider v3.0.28 (#166)
+- Upgraded to Unity 2022.3.27f1 (#170)
```

Comment:
As sub-bullets, we should highlight some of the services and packages like Transport that were updated.
So items like Authentication, qos, rendering, Transport.